src/community/ogcapi/ogcapi-features/src/test/java/org/geoserver/ogcapi/v1/features/CollectionsTest.java

```diff
@@ -67,6 +72,32 @@ public void testCollectionsJson() throws Exception {
         testCollectionsJson(json);
     }
 
+    @Test
+    public void testSkipMisconfigured() throws Exception {
+        // enable skipping of misconfigured layers
+        GeoServerInfo global = getGeoServer().getGlobal();
+        global.setResourceErrorHandling(ResourceErrorHandling.SKIP_MISCONFIGURED_LAYERS);
+        getGeoServer().save(global);
+        // manually misconfigure one layer
+        FeatureTypeInfo misconfigured =
```

Comment:
At this point you're making it recognizable, but you're not really misconfiguring it, just making it recognizable... am I misreading this?

Reply:
Fixed, removed unnecessary changes